uwp/Add-Syncfusion-Controls.md

```diff
@@ -160,7 +160,7 @@ The following steps direct you to create the **Syncfusion** **UWP** **Applicatio
 
 1. To create a Syncfusion UWP project, choose **New** **Project****->** **Syncfusion****->****Universal** **Windows****->****Syncfusion** **Universal** **Windows** **Application** from **Visual** **Studio**.
 
-   ![Create Syncfusion UWP project](Add-Syncfusion-Controls_images/Syncfusion-Project-Templates_img1.jpeg)
+   ![Create Syncfusion UWP project](add-syncfusion-controls_images/uwp-add-syncfusion-controls-project-templates.jpeg)
```

Comment:
Remove the add-syncfusion-controls in all places.
Update the proper image name for that images.

Reply:
I have updated proper image name